tasks/transform_conformance/tests/babel-plugin-transform-typescript/test/fixtures/class-export/input.js

```diff
@@ -0,0 +1,2 @@
+class Foo {}
+export { Foo };
```

Comment:
Should we have a test including all declarations? `class`, `function`, `interface` etc

Reply:
Done